components/language-switcher/readme.md

```diff
@@ -0,0 +1,8 @@
+A UI molecule for changing the language of the current page.
```

Comment:
We should phase out the atomic design jargon and just call everything a "component."

Reply:
Oh, are we completely ditching the atomic design lingo? Guessing all the components will be under one header then?

Reply:
Yeah... atomic lingo is very useful for discussing design systems but I'm not sure it's useful as an information architecture strategy.